contracts/contracts/timelock/Timelock.sol

```diff
@@ -159,7 +159,7 @@ contract Timelock {
         string memory signature,
         bytes memory data,
         uint256 eta
-    ) public payable returns (bytes memory) {
```

Comment:
We should remove the `values` from the proposal in the Governor and the payable modifier from the Governor's execute too I think?

Reply:
Done. PTAL...

Reply:
💙

Reply:
BTW, I think we can also remove `function() external payable {}` from Timelock.sol?
https://github.com/OriginProtocol/origin-dollar/blob/master/contracts/contracts/timelock/Timelock.sol#L70